server/views/settings.ejs

```diff
@@ -1,5 +1,5 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
```

Comment:
And this?

Reply:
It has `<!DOCTYPE html>` and `html` tag. If we use Bootstrap and stuff we are surely with HTML5, so it's better to have headers according to HML5 standards, not HTML4 ones.